### ISSUE TITLE
Type Password Again is Missing the Blue Focus Outline

### ISSUE THREAD

**Issue by samuraininja360** [NinjaSpect/ninjaspect.github.io#11]
@TheBugCoder I think you forgot to change the JavaScript that controls the Outlines once you click an input field

`:focus` is involved in the modal by the way!
